libs/vscode/nx-conversion/src/lib/vscode-nx-conversion.ts

```diff
@@ -0,0 +1,139 @@
+import { getShellExecutionForConfig, getTelemetry } from '@nx-console/server';
+import { WorkspaceConfigurationStore } from '@nx-console/vscode/configuration';
+import { pipe, Subject } from 'rxjs';
+import { filter, scan, tap } from 'rxjs/operators';
+import {
+  commands,
+  ExtensionContext,
+  Task,
+  tasks,
+  TaskScope,
+  window,
+} from 'vscode';
+
+const PROMPT_COUNT = 1;
+const PROMPT_MSG = 'Convert to Nx?';
```

Comment:
I was thinking for more along the lines of "Would you like to upgrade the Angular CLI with Nx (making your `ng` commands up to 10 times faster)?"

Reply:
Yup, we'll need to think of a good tag line